store/src/main/java/com/issoft/store/ProductNameComparator.java

```diff
@@ -0,0 +1,45 @@
+package com.issoft.store;
+
+import org.w3c.dom.Document;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.File;
+import java.io.IOException;
+import java.util.Comparator;
+
+public class ProductNameComparator implements Comparator<Product>{
+
+    @Override
+    public int compare(Product firstProduct, Product secondProduct) {
+
+        File file = new File("C:\\Users\\katyagolovko\\OnlineStoreJavaTraining\\consoleApp\\src\\main\\resources\\store.xml");
```

Comment:
absolute path is not good to set. Other person cannot execute your code. Such as I :)
the good practice is use relative path. ".\\consoleApp\\src\\main\\resources\\store.xml"
Also it is good to use backward slash as it allow to execute code on linus or Mac Os operation systems
"./consoleApp/src/main/resources/store.xml"